src/parse/asp/builtins.go

```diff
@@ -279,6 +279,8 @@ func isType(obj pyObject, name string) bool {
 		return name == "list"
 	case pyDict:
 		return name == "dict"
+	case *pyConfig:
+		return name == "CONFIG"
```

Comment:
possibly this should be "config"... the variable is capitalised since it's "constant" but the type would be more consistent as lower-case.